### PR TITLE
Bugfix/process images in gatsby node bug

### PR DESCRIPTION
**Pull request by thiagodebastos**
### TLDR;
This was a bit convoluted. I fixed an issue where gatsby would fail to build when I tried to access the bannerImage graphql node from within `gatsby-node.js`. It looks like I need a better understanding of 
what goes on in there.

### What I was trying to do
Add a banner image to each blog post from within the `blog-post.js` template, if one exists in the post's frontmatter.

### What was happening
Gatbsy would fail to build since `graphql` would think that `bannerImage` is of type `String`, and not `File`.

### What I did to fix it
I removed the call to `bannerImage` as a `File` from `gatsby-node.js`.

### Takeaways
Learn more about how Gatsby utilised graphql.

### CODE REVIEW

**Comment by thiagodebastos** [comment]
So it built fine, then it failed on Netlify. So I checked it locally again and got the problem back!

```bash
error GraphQL Error Field "banner" must not have a selection since type "String" has no subfields.

  file: /Users/tdavoodifar/Documents/dev/repos/thiago.codes/src/templates/blog-post.js

   1 |
   2 |   query blogPostQuery($id: String!) {
   3 |     markdownRemark(id: { eq: $id }) {
   4 |       html
   5 |       fields {
   6 |         title
   7 |         date
   8 |         author
>  9 |         banner {
     |                ^
  10 |           childImageSharp {
  11 |             fluid(maxWidth: 1000) {
  12 |               ...GatsbyImageSharpFluid_withWebp_tracedSVG
  13 |             }
  14 |           }
  15 |         }
  16 |       }
  17 |     }
  18 |   }
  19 |

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
